catalog_full_include.html

```diff
@@ -0,0 +1,27 @@
+{% for c in core %}
+    <h3 style="height:5px">{{c['name']}}</h3>
+    <b>Schedule: </b>
+    {{ c['schedule'] | join(' / ', attribute='daypart') }}
+{% endfor %}
+<br>
+{% for c in classes %}
+  {% if not c['exclude_from_catalog'] %}
```

Comment:
Is there a {% continue %} statement in jinja2?  I like to reduce nesting whenever possible. So instead we can write {% if c['exclude_from_catalog'] %} {% continue %}.